db/seeds/challenge/challenge.rb

```diff
@@ -33,7 +33,51 @@
   comment_placeholder: 'Belonging to me is...',
   starts_at: Time.now,
   ends_at: 1.month.from_now,
-  featured: true
+  featured: true,
+  drawing: "<div class='legal container'>
```

Comment:
If possible, can we have the the `drawing` page actually have the `container`, `row`, and `col-md-10` elements? That way, we only have to define the actual terms in HTML, but don't worry about how they are displayed on the page in terms of layout / containers?